core/src/game_state.rs

```diff
@@ -296,6 +311,22 @@ impl PropagatedState {
         }))
     }
 
+    pub fn set_friend_selection_policy(
```

Comment:
Let's take `policy: FriendSelectionPolicy` directly -- serde should handle the translation for us

Reply:
done